sdk/core/core-client/src/urlHelpers.ts

```diff
@@ -169,13 +169,10 @@ function calculateQueryParameters(
           queryParameterValue = queryParameterValue.join(delimiter);
         }
 
-        // ignore empty values
-        if (queryParameterValue) {
-          result.set(
-            queryParameter.mapper.serializedName || getPathStringFromParameter(queryParameter),
-            queryParameterValue
-          );
-        }
+        result.set(
```

Comment:
I honestly don't remember why the if check was here. If the live tests for tables and event grid pass with this change, I suppose I'm not too worried about it.

Reply:
Both are passing in the checks.